apps/api/src/controllers/competition.controller.ts

```diff
@@ -495,6 +495,10 @@ export function makeCompetitionController(services: ServiceRegistry) {
             pagingParams,
           );
 
+        // Populate the rewards for each competition
+        // const competitionsWithRewards =
+        //   await services.coreRewardService.populateRewards(competitions);
+
```

Comment:
need to remove this. It was a temp solution to add the rewards to the competitions response, but I replaced this in favor of a join to the rewards table.